internal/models/policy/recipe/network/allow_all.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+Code generated by go-swagger; DO NOT EDIT.
+*/
+
+package policyrecipenetworkmodel
```

Comment:
The models have been removed, not sure why they were coming in this PR, I manually removed the directory, since it is already present in the feature branch.